main.cpp

```diff
@@ -371,6 +439,8 @@ struct Laundromat
     int addCoinDispenser(int numberOfDispensers = 1, std::string coinType = "Quarter");
     // increase number of employees
     int hireEmployee(int staffTotal, std::string employeeName, bool nowHiring = true);
+    // new member function
+    Laundromat getNewLaundromat(int threshold, int startingValue);
```

Comment:
a Laundromat instance will return another Laundromat?
that doesn't make much sense.
This is a member function, so the only way to call it is by using a Laundromat instance...
- [ ] revise